frontend/src/Components/SignUp/SignUp.js

```diff
@@ -0,0 +1,41 @@
+import { useCallback } from "react";
+import React from "react";
+import { withRouter } from "react-router";
```

Comment:
FYI, since v4 `react-router` can be used with hooks instead HOC (Higher Order Components), so instead wrapping each component needed with router you can simply use a `useHistory`, `useParams`, `useRouter`, `useLocation` based on your needs.

https://reactrouter.com/web/api/Hooks